public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -351,9 +355,13 @@ export const ReceiversTable = ({ config, alertManagerName }: Props) => {
   );
 
   const [createSupported, createAllowed] = useAlertmanagerAbility(AlertmanagerAction.CreateContactPoint);
+  const canReadSecrets = contextSrv.hasPermission(
```

Comment:
Could we use `useAlertmanagerAbility` for `canReadSecrets`?

Reply:
we don't have an specific AlertmanagerAction for this case, maybe we could add it?

Reply:
done!